mvx-esdt-safe/tests/mvx_esdt_safe_blackbox_tests.rs

```diff
@@ -252,7 +252,7 @@ fn test_register_token_not_enough_egld() {
 
     let token_hash = register_token_args.generate_hash();
     let hash_of_hashes = ManagedBuffer::from(&sha256(&token_hash.to_vec()));
-    let bitmap = ManagedBuffer::new_from_bytes(&[1]);
+    let bitmap = ManagedBuffer::new_from_bytes(&[0x01, 0x02]);
```

Comment:
```suggestion
    let bitmap = ManagedBuffer::new_from_bytes(&[0x01]);
```
This and some more below